src/node_env_var.cc

```diff
@@ -66,89 +66,72 @@ std::shared_ptr<KVStore> system_environment = std::make_shared<RealEnvStore>();
 Local<String> RealEnvStore::Get(Isolate* isolate,
```

Comment:
@addaleax Since we have been throwing exceptions here ...shouldn't this be `MaybeLocal` and gets handled by the caller?

Reply:
Yeah, this should probably start to return `MaybeLocal`, throw exceptions in those cases, and update the callers to handle those…